tests/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_executable( chain_test ${UNIT_TESTS} ${COMMON_SOURCES} )
 target_link_libraries( chain_test eos_native_contract eos_chain chainbase eos_utilities eos_egenesis_none wallet_plugin fc ${PLATFORM_SPECIFIC_LIBS} )
 
 if(WASM_TOOLCHAIN)
+  add_subdirectory(slow_tests/memory_test)
```

Comment:
This should not be necessary once the memory test contract is moved to the contracts tree.

Reply:
See my previous response.